ddl/ddl_api.go

```diff
@@ -1789,6 +1789,11 @@ func (d *ddl) RebaseAutoID(ctx sessionctx.Context, ident ast.Ident, newBase int6
 	if err != nil {
 		return errors.Trace(err)
 	}
+	// If newBase < autoIncID, we can't do nothing and just return nil.
+	// If has 2 TiDB-server, TiDB-A: 0 ~ 30000, TiDB-B: 30001 ~ 60000,
+	// if user send SQL `alter table t1 auto_increment = 100` to TiDB-B,
+	// and TiDB-B do nothing and just return here,
+	// then TiDB-A is possible to allocate 99 for auto_increment column, this is not reasonable for the user.
```

Comment:
I can not quite understand this comment. Could you please add a test case for it, or refine the comment?

Reply:
This comment just explains the code in the below. Because sometimes when I review this code, I think we can simply return here in some case, but after careful consideration, I found below code is reasonable, so I add this comment.